app.py

```diff
@@ -36,38 +35,65 @@
 with app.app_context():
     app.config.from_pyfile('flask.cfg')
 
-# Add theme loader.
-theme_path = path.join(app.config.get('YODA_THEME_PATH'), app.config.get('YODA_THEME'))
-theme_loader = ChoiceLoader([
-    FileSystemLoader(theme_path),
-    app.jinja_loader,
-])
-app.jinja_loader = theme_loader
 
-
-# Load banner configurations
-def load_banner_config():
-    """Load or initialize banner configurations."""
-    config_file_path = path.join(app.config['APP_SHARED_FOLDER'], 'banner_settings.json')
-    default_config = {'banner_enabled': False}
+def load_admin_config():
+    """Load or initialize admin configurations from config file, writing defaults if no config file exists."""
+    config_file_path = path.join(app.config['APP_SHARED_FOLDER'], 'admin_settings.json')
+    def_config = {
+        'banner': {
+            'banner_enabled': False,
+            'banner_importance': False,
+            'banner_message': ''
+        },
+        'YODA_THEME': app.config.get('YODA_THEME')
```

Comment:
The "YODA_THEME" is used here for updating the same variable in the app.config. Replacing with "theme" would result in fail in theme reading. 

Since the "YODA_THEME" is no longer a constant. For consistency,  I'd suggest to use "theme" across the flask configuration and Ansible playbook. What do you think?